core/src/main/java/com/netflix/conductor/core/execution/tasks/Join.java

```diff
@@ -126,6 +126,6 @@ public Optional<Long> getEvaluationOffset(TaskModel taskModel, long defaultOffse
     }
 
     public boolean isAsync() {
-        return false;
+        return true;
```

Comment:
the change made here https://github.com/conductor-oss/conductor/pull/120

the result was simple joins are very slow +30secs and more complex joins take +40mins